languages/wikipron/src/serbo_croat_split.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import re
+import os
+import logging
+
+from codes import SERB_CROAT_PATH
+
+
+def _contains_latin(word):
+    return bool(re.search("[\u0000-\u007F]", word))
```

Comment:
I think this check is too weak. Here's what I'd suggest. 

`_all_latin` is true iff *every* character in a word is ASCII. 
`_all_cyrillic` is true iff *every* character in a word is Cyrillic (have to allow for dash, which can at least be used in some other Cyrillic scripts). 

If a word doesn't pass one or the other, log it and move on.